pkg/arvo/gen/hood/ames-snub.hoon

```diff
@@ -5,6 +5,6 @@
 ::::
   ::
 :-  %say
-|=  [^ ships=(list ship) ~]
+|=  [^ [form=?(%allow %deny) ships=(list ship)] ~]
 :-  %helm-ames-snub
-ships
+[form ships]
```

Comment:
this should have a trailing newline, for consistency with the rest of the codebase